server/catgenome/src/main/java/com/epam/catgenome/manager/externaldb/ncbi/NCBIDataManager.java

```diff
@@ -59,8 +61,15 @@
     protected static final String NCBI_SEARCH = "entrez/eutils/esearch.fcgi?";
     protected static final String NCBI_LINK = "entrez/eutils/elink.fcgi?";
     protected static final String RETMODE_PARAM = "retmode";
+    protected static final Integer MAX_RESULTS_PARAM_VALUE = 500;
```

Comment:
I'd suggest to move this param to `catgenome.properties` to allow customisation and set default value to 100.

Reply:
Agree, fixed. Now controlled by `externaldb.ncbi.max.results` property, default to 100 and also added startup check to verify it's not more that max allowed 500.